cmd/grr/workflow.go

```diff
@@ -71,6 +72,23 @@ func pullCmd() *cli.Command {
 	}
 	return initialiseCmd(cmd, &opts)
 }
+
+func pullDashboardCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "pull-dashboard <resource-path>",
+		Short: "Pulls remote dashboards and writes them to local source in JSON format",
+		Args:  cli.ArgsExact(1),
+	}
+	opts := grizzly.Opts{
+		Legacy: true,
+	}
+
+	cmd.Run = func(cmd *cli.Command, args []string) error {
+		return grizzly.Pull(args[0], opts)
```

Comment:
Wouldn't this pull more resources and not only dashboards? I feel like this piece of code should manipulate `grizzly.Opts.Targets` before calling `grizzly.Pull` to get the expected behavior. 

Note that the user may also provide it, to refer specific dashboards.

Reply:
Perhaps using `targetsOfKind` also here? 🤔

Reply:
Yes-yes, good catch!